gui/tasks/distribution.js

```diff
@@ -304,6 +320,21 @@ function packLinux() {
     targets: builder.Platform.LINUX.createTarget(),
     config: {
       ...config,
+      beforeBuild: (options) => {
+        switch (options.arch) {
+          case 'x64':
+            process.env.TARGET_TRIPLE = 'x86_64-unknown-linux-gnu';
```

Comment:
`env.TARGET_TRIPLE` doesn't seem to be used anywhere for Linux? I wouldn't mind using it as on macOS though.

Also would it be possible to extract this code and the very similar code for macOS to a function used for both?